src/api/route-services/experiment.js

```diff
@@ -122,10 +126,16 @@ class ExperimentService {
       lastViewed: body.lastViewed,
       projectId: body.projectUuid || body.projectId,
       description: body.description,
-      meta: body.meta,
-      processingConfig: body.processingConfig,
     };
 
+    if (body.meta) {
+      this.updatePropertyFromDiff(experimentId, 'meta', toUpdatePropertyArray(body.meta));
+    }
+
+    if (body.processingConfig) {
+      this.updateProcessingConfig(experimentId, toUpdatePropertyArray(body.processingConfig));
+    }
+
```

Comment:
Is there any reason why each of these properties are updated individually? Can't they be updated together inside the `params` below?

Reply:
I just went with the simpler solution but you're right, we should actually update them the other way